CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog (master)
 
+### v5.0.1
+* Fix: Upgrade Jest to 22.1.x
```

Comment:
How about:
Chore: Upgrade to Jest 22.1.x internally

Reply:
no problem :)